configs/openSUSE/pam-modules.toml

```diff
@@ -421,10 +421,10 @@ nodigests = [
 ]
 
 [[FileDigestGroup]]
-package = "pam_kwallet"
+packages = ["pam_kwallet", "pam_kwallet6"]
 type = "pam"
 note = "imported from rpmlint1 PAMModules.WhiteList"
-bug = "bsc#993806"
+bugs = ["bsc#993806", "bsc#1217183"]
 nodigests = [
     "glob:*/security/pam_kwallet5.so",
```

Comment:
Ok

Very strange that they still use the *5 names for kde6 components

Reply:
I mentioned that explicitly for the PAM module but nobody bothered.